scss/forms/_quantity-selector.scss

```diff
@@ -33,6 +33,13 @@
       padding-right: $quantity-selector-btn-padding-x;
       border-right: none;
 
+      &:focus {
+        &[data-focus-visible-added] { // stylelint-disable-line selector-no-qualifying-type
+          margin-right: calc(#{$input-border-width} * -1); // stylelint-disable-line function-disallowed-list
+          border-right: var(--#{$prefix}border-width) solid $gray-500;
```

Comment:
Can't we play with `border-width` instead of `border` to avoid using an `!important` in the `_forms.scss` file (same l.34, 52 and 58) ?